Makefile.toml

```
@@ -511,6 +524,38 @@ cargo install \
 '''
 ]
 
+[tasks.check-infra-lock]
+dependencies = ["infra-tools"]
+script_runner = "bash"
+script = [
+'''
+set -e
+
+export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+
+infrasys \
+   --infra-config-path "${PUBLISH_INFRA_CONFIG_PATH}" \
+   \
+   check-infra-lock 
+'''
+]
+
+[tasks.create-infra]
+dependencies = ["check-infra-lock"]
```

As we have been discussing elsewhere, if we remove the check-infra-lock subcommand then this task is the one that depends on `setup`.

Does `setup` ensure the installation of `tuftool`?

```suggestion
dependencies = ["check-infra-lock", "setup"]
```

---

Made it dependent on infra-tools which is dependent on setup (added tuftool to infra-tools!)